File: implementation/WCF/WCF_USERSERVICE/UserCreationRequest.cs
namespace WCF_USERSERVICE
{
    
    
    /// <remarks/>
    [System.CodeDom.Compiler.GeneratedCodeAttribute("System.Xml", "4.0.30319.233")]
    [System.SerializableAttribute()]
    [System.Diagnostics.DebuggerStepThroughAttribute()]
    [System.ComponentModel.DesignerCategoryAttribute("code")]
    [System.Xml.Serialization.XmlTypeAttribute(AnonymousType=true, Namespace="http://localhost:9000/userservice.xsd")]
    public partial class UserCreationRequest
    {
        
        /// <remarks/>
        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
        public string username;
        
        /// <remarks/>
        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
        public string firstname;
        
        /// <remarks/>
        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
        public string lastname;
        
        /// <remarks/>
        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
        public bool gender;
        
        /// <remarks/>
        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified, DataType="integer")]
        public string age;
    }
}

File: implementation/PHP/UserCreationRequest.php
<?php

class UserCreationRequest
{

  /**
   * 
   * @var string $username
   * @access public
   */
  public $username;

  /**
   * 
   * @var string $firstname
   * @access public
   */
  public $firstname;

  /**
   * 
   * @var string $lastname
   * @access public
   */
  public $lastname;

  /**
   * 
   * @var boolean $gender
   * @access public
   */
  public $gender;

  /**
   * 
   * @var int $age
   * @access public
   */
  public $age;

  /**
   * 
   * @param string $username
   * @param string $firstname
   * @param string $lastname
   * @param boolean $gender
   * @param int $age
   * @access public
   */
  public function __construct($username, $firstname, $lastname, $gender, $age)
  {
    $this->username = $username;
    $this->firstname = $firstname;
    $this->lastname = $lastname;
    $this->gender = $gender;
    $this->age = $age;
  }

}
